score-rce/src/main/java/io/cloudslang/content/remote/constants/Constants.java

```diff
@@ -0,0 +1,23 @@
+package io.cloudslang.content.remote.constants;
+
+/**
+ * Created by pasternd on 08/06/2016.
+ */
+public class Constants {
+
+    public static final String WMI = "wmic /node:\"";
+    public static final String CREATE_FILE_COM = " process call create \"cmd.exe fsutil file createnew ";
+    public static final String CALL_CMD_COM = "\" process call create \"cmd.exe /c ";
+    public static final String USER = "\" /user:" ;
+    public static final String PASSWORD = " /password:";
+    public static final String BACK_SLASH = " \"";
+    public static final String END_FILE_COM = " 0\"";
+    public static final String EMPTY_STR = "";
+    public static final String START_PATH_FILE = "\\\\";
+    public static final String FILENAME = "\\C$\\Temp\\emptyfile.txt";
```

Comment:
What will happen if 2 instances of this operation will run in the same time (in parallel) on the same host ?
Quick hint: the results for both operation will be mixed (messed up). 
"\\C$\\Temp\\emptyfile" + Math.random() +".txt"; would be more appropriate.